### PR TITLE
Address #3943

### PR DESCRIPTION
**Pull request by clusterfack**
Removes the cyborg hand consistency check from mommis when preparing to ventcrawl at least.

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -286,7 +286,7 @@
 						src << "<span class='warning'>You didn't choose anything.</span>"
 						return
 
-					if(!do_after(src, 45))
+					if(!do_after(src, 45,,0))
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/carbon.dm]: Rather than doing this, you should just name the argument.

Comment (clusterfack) [inline code/modules/mob/living/carbon/carbon.dm]: This practice is just as commonly used

Comment (ComicIronic) [inline code/modules/mob/living/carbon/carbon.dm]: It's less likely to be caught if someone changes the arg order, though.

Comment (d3athrow) [inline code/modules/mob/living/carbon/carbon.dm]: the other method is just as likely to cause runtimes if somebody changes the arg name for some reason.

**Pull request merged**